pkg/vm/vm.go

```diff
@@ -736,6 +736,15 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 			panic(err.Error())
 		}
 
+	case opcode.REVERSE3, opcode.REVERSE4, opcode.REVERSEN:
+		n := 3 + int(op-opcode.REVERSE3)
```

Comment:
I'd rather not obfuscate `REVERSE4` here this way.

Reply:
Fixed.